pkg/sysinfo/sysinfo_linux.go

```diff
@@ -62,6 +62,11 @@ func New(quiet bool) *SysInfo {
 		}
 	}
 
+	// Check if cgroup namespaces are supported.
+	if _, err := os.Stat("/proc/self/ns/cgroup"); !os.IsNotExist(err) {
+		sysInfo.CgroupNamespaces = true
```

Comment:
We may want to extract it to a function, e.g.

```suggestion
		sysInfo.CgroupNamespaces = cgroupNamespacesSupported()
```

Looking at that, perhaps this should be a utility in libcontainer; somewhere around here; https://github.com/opencontainers/runc/blob/f5b99917df9fbe1d9a4114966fb088dd6860e85a/libcontainer/cgroups/utils.go#L30-L45

and/or in the containerd/cgroups package https://github.com/containerd/cgroups/blob/master/utils.go

Reply:
Good idea - it makes sense to me for this to live in libcontainer.  What do you think about adding it in its own sub-package of libcontainer?  Everything in the `libcontainer/cgroups` package deals with cgroups themselves, whereas this is actually a namespace - I worry that putting them together in the cgroups package might make this more confusing.

So maybe something like `libcontainer/namespace/cgroups.go`?